adapters/restclient/lib/opentelemetry/adapters/restclient/adapter.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Adapters
+    module RestClient
+      class Adapter < OpenTelemetry::Instrumentation::Adapter
+        install do |config|
+          require_dependencies
+          config[:request_patch] ||= Patches::Request
+          patch_request
+        end
+
+        present do
+          Gem.loaded_specs.include?('rest-client')
```

Comment:
We should actually be checking `defined?(::RestClient)`. The reason - a gemspec can be loaded, but the gem can go unrequired. When that happens we reference an undefined constant. The instrumentation installation handles this gracefully, but dumps a nasty stack trace in the logs. We should check for constants in the target library instead. Sorry for setting a bad example earlier on. I've made the same changes to Faraday and Sinatra in #177.